publication-commons/src/main/java/no/unit/nva/publication/model/business/importcandidate/ImportEntityDescription.java

```diff
@@ -0,0 +1,16 @@
+package no.unit.nva.publication.model.business.importcandidate;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+import no.unit.nva.model.PublicationDate;
+import no.unit.nva.model.Reference;
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
+public record ImportEntityDescription(String mainTitle, URI language, PublicationDate publicationDate,
+                                      List<ImportContributor> contributors, String mainAbstract,
+                                      Map<String, String> alternativeAbstracts, List<String> tags, String description,
```

Comment:
```suggestion
                                      Collection<ImportContributor> contributors, String mainAbstract,
                                      Map<String, String> alternativeAbstracts, Collection<String> tags, String description,
```

Reply:
We probably need to update PMD rule set. But in next pr.